### PR TITLE
Fix executable user default functions execution with Nullable arguments

### LINKED ISSUE #35897: user-defined-functions null value input
When I'm executing a UDF function, a `null value` input to the script is received as `0`(for numbers) and `empty string` (for strings). 

The UDF definition like this
```
<function>
    <type>executable</type>
    <name>some_function_name</name>
    <return_name>result</return_name>
    <return_type>String</return_type>
    <argument>
        <type>Nullable(Int32)</type>
        <name>argument_1</name>
    </argument>
    <format>JSONEachRow</format>
    <command>some_function.py</command>
</function>
```
And the script file link this (Here whatever input I got I'm logging it to a file.)
```
#!/usr/bin/python3

import sys
import json

def write_to_file(line):
	with open("/tmp/input.log",'a') as fw:
		fw.write(str(line))

if __name__ == '__main__':
    for line in sys.stdin:
        # Writing log
        write_to_file(line)

        result = {"result":str(line)}
        print(json.dumps(result))
        sys.stdout.flush()
```

I got the below results 
```
clickhouse-client --query="select value, some_function_name(value) from (SELECT 1 as value UNION ALL SELECT NULL UNION ALL SELECT 0) FORMAT Vertical"  --input_format_tsv_empty_as_default=1

Row 1:
──────
value:                     1
some_function_name(value): {"argument_1":1}


Row 2:
──────
value:                     ᴺᵁᴸᴸ
some_function_name(value): ᴺᵁᴸᴸ

Row 3:
──────
value:                     0
some_function_name(value): {"argument_1":0}

```
Here, the result for `Row 2` should be `{"argument_1":NULL}` but it's `NULL`.

And the log I got  for inputs as below
```
{"argument_1":1}
{"argument_1":0}
{"argument_1":0}
```

So, is there any option to get the `NULL` value as `NULL` to the input script?

### PR DESCRIPTION
**Pull request by kitaisreal**
### Changelog category (leave one):
- Bug Fix (user-visible misbehavior in official stable or prestable release)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Fix formatting of Nullable arguments for executable user defined functions. Closes #35897.

### CODE REVIEW

**Comment by kitaisreal** [comment]
@Mergifyio update

File: tests/integration/test_executable_user_defined_function/test.py [hunk 1]
```diff
@@ -228,3 +228,15 @@ def test_executable_function_sum_json_python(started_cluster):
     )
 
     node.query("DROP TABLE test_table;")
+
+def test_executable_function_input_nullable_python(started_cluster):
+    skip_test_msan(node)
```
Comment (vdimir) [inline tests/integration/test_executable_user_defined_function/test.py:233]: Why do we skip msan for all tests? Just because of speed?

**Review by vdimir — APPROVED** [review]

File: tests/integration/test_executable_user_defined_function/test.py (hunk 1 above)
Comment (kitaisreal) [reply tests/integration/test_executable_user_defined_function/test.py:233]: It has problems with `vfork` system call.

File: tests/integration/test_executable_user_defined_function/functions/test_function_config.xml [hunk 2]
```diff
@@ -289,4 +289,26 @@
         <command>input_sum_json_named_args.py</command>
     </function>
 
+    <function>
+        <type>executable</type>
+        <name>test_function_nullable_python</name>
```
[the remark this answers is not in the document]
Comment (kitaisreal) [reply tests/integration/test_executable_user_defined_function/functions/test_function_config.xml:294]: Need to rename type to `executable_pool`. Fixed.

**Pull request merged**